components/webdriver_server/lib.rs

```diff
@@ -518,7 +518,7 @@ impl Handler {
 
     fn focus_webview_id(&self) -> WebDriverResult<WebViewId> {
```

Comment:
I think this method has been accidentally misnamed. `focus_webview_id` is a request to focus a WebView, but `focused_webview_id` is a getter to retrieve the currently focused `WebViewId`.
```suggestion
    fn focus_webview_id(&self) -> WebDriverResult<WebViewId> {
```